src/ast/irbuilderbpf.cpp

```diff
@@ -1223,8 +1223,7 @@ void IRBuilderBPF::CreateCheckSetRecursion(const Location &loc,
   // Most of the time this will happen for the functions that can lead
   // to a crash e.g. "queued_spin_lock_slowpath" but it can also happen
   // for nested probes e.g. "page_fault_user" -> "print".
-  CreateAtomicIncCounter(to_string(MapType::EventLossCounter),
-                         bpftrace::BPFtrace::event_loss_cnt_key_);
+  CreateAtomicIncCounter();
```

Comment:
Seems like this function is no longer generic so perhaps this is a better name.
```suggestion
  CreateIncEventLossCounter();
```

Reply:
Done